addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -79,6 +79,48 @@
     animation: catchAttention 200ms ease 0s infinite normal;
 }
 
+// ----------------------------------------------------------------------------
+// Render a "tree-view" design on a set of vertical elements
+
+// @param {number} --treeEntry-padding-h - Entry's horizzontal padding.
+// @param {number} --treeEntry-padding-v - Entry's vertical padding.
+// @param {number} --treeEntry--before-top - Vertical-line top position.
+// @param {string} --treeEntry--after-display - Horizzontal-line display mode.
+// @param {color}  --treeEntry--beforeAfter-color - Lines color
+// @param {color}  --treeEntry--beforeAfter-left - Lines left position
+// ----------------------------------------------------------------------------
+.o_treeEntry {
```

Comment:
Naming remark: why use camelCase + `o_` prefix instead of the usual snake case (`o_tree_entry`) ? :thinking: